Description: fix typo in courier plugin. Fixed upstream in revision 1174, and
 in tags/1.3.1
Origin: other, http://bugs.debian.org/491463
Bug-Debian: http://bugs.debian.org/491463
Index: munin-1.2.6.obsolete.0.945208103469671/node/node.d/courier_.in
===================================================================
--- munin-1.2.6.obsolete.0.945208103469671.orig/node/node.d/courier_.in	2008-07-21 00:24:10.000000000 +0200
+++ munin-1.2.6.obsolete.0.945208103469671/node/node.d/courier_.in	2008-07-21 00:24:16.000000000 +0200
@@ -26,7 +26,6 @@
 #
 # courierpop3login:
 # Jan 22 06:28:24 raven courierpop3login: Connection, ip=[::ffff:192.168.0.1]
-5B
 # Jan 22 06:48:22 raven courierpop3login: DISCONNECTED, user=someuser, ip=[::ffff:192.168.0.1], top=0, retr=0, time=21
 # Jan 22 06:28:24 raven courierpop3login: LOGIN, user=someuser, ip=[::ffff:192.168.0.1]
 # Jan 22 06:28:25 raven courierpop3login: LOGOUT, user=someuser, ip=[::ffff:192.168.0.1], top=0, retr=0, time=0
